models/public/hrnet-w32-human-pose-estimation/model.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from config import cfg
+import models
+import torch
```

Comment:
common rule for order python modules import:
1. built-in packages (os, pathlib, sys, json e.t.c.)
2. installed packages (torch, numpy, cv2, e.t.c.)
3. local packages from your project
so
```suggestion
import torch
from config import cfg
import models
```